core/experiments/manager.php

```diff
@@ -448,7 +471,9 @@ private function render_feature_settings_field( array $feature ) {
 				<?php } ?>
 			</select>
 			<p class="description"><?php echo $feature['description']; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?></p>
-			<div class="e-experiment__status"><?php echo sprintf( esc_html__( 'Status: %s', 'elementor' ), $this->release_statuses[ $feature['release_status'] ] );  // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?></div>
+			<?php if ( 'stable' !== $feature['release_status'] ) { ?>
+				<div class="e-experiment__status"><?php echo sprintf( esc_html__( 'Status: %s', 'elementor' ), $this->release_statuses[ $feature['release_status'] ] );  // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?></div>
```

Comment:
@guyc-E Instead of the 'phpcs' comment, use: `Utils::print_unescaped_internal_string();`